File: 240110/함수를 이용한 윤년 판별/tell-the-function-using-a-leap-year.py
y = int(input())
def yun(n):

    if n%4 !=0:
        return False
    elif n %100 !=0:
        return True
    elif n%400 !=0:
        return False
    else:
        return True

if yun(y):
    print("true")
else:
    print("false")

File: 240110/함수를 이용한 윤년 판별/tell-the-function-using-a-leap-year.java
import java.util.Scanner;
public class Main {
    public static boolean yun(int a){
        if (a%4==0){
            return true;
        }
        else if( a% 100 == 0){
            return false;
        }
        else if(a%400==0){
            return true;
        }
        else{
            return false;
        }
    }
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        System.out.print(yun(n));
    }
}